src/pages/Defi/hooks/useFoxyBalances.tsx

```diff
@@ -22,9 +22,11 @@ export function useFoxyBalances() {
 
   useEffect(() => {
     ;(async () => {
-      const chainAdapter = await getChainAdapterManager().get(KnownChainIds.EthereumMainnet)
+      const chainAdapter = getChainAdapterManager().get(KnownChainIds.EthereumMainnet)
       if (!chainAdapter || !wallet) return
-      const userAddress = await chainAdapter.getAddress({ wallet })
+      // TODO accountNumber needs to come from account metadata
+      const bip44Params = chainAdapter.getBIP44Params({ accountNumber: 0 })
```

Comment:
@gomesalexandre curious on your input here

Reply:
Yeah we should just be able to pass one / all accountIds depending on if this returns balances for an account (as it does currently) or a mapping of accountIds to balances.

I'm fine with this as this is an improvement using bip44Params, but we shouldn't use the chain-adapter to get them, we have selectors that do precisely this

Reply:
Captured in https://github.com/shapeshift/web/issues/2730